File: Protótipos(back-end)/projetoFila(java)/Cliente.java
package projetoMascate;

public class Cliente {
    private String nome;
    private int senha;

    public Cliente(String nome, int senha) {
        if (nome.length() > 10) {
            throw new IllegalArgumentException("O nome não pode ter mais que 10 caracteres.");
        }
        this.nome = nome;
        this.senha = senha;
    }

    public String getNome() {
        return nome;
    }

    public int getSenha() {
        return senha;
    }
}

File: Protótipos(back-end)/projetoFila(ts)/Cliente.ts
class Cliente {
    private nome: string;
    private senha: number;

    constructor(nome: string, senha: number) {
        if (nome.length > 10) {
            throw new Error("O nome não pode ter mais que 10 caracteres.");
        }
        this.nome = nome;
        this.senha = senha;
    }

    getNome(): string {
        return this.nome;
    }

    getSenha(): number {
        return this.senha;
    }
}

export default Cliente;
